Source/DataSources/GeoJsonDataSource.js

```diff
@@ -604,6 +622,20 @@ define([
                 defaultFillMaterialProperty = new ColorMaterialProperty(defaultFill);
             }
         },
+        /**
+         * Gets or sets default of whether to draw on terrain.
+         * @memberof GeoJsonDataSource
+         * @type {Boolean}
+         * @default false
+         */
+        terrain : {
```

Comment:
Are we sure `terrain` is the best name for this property?  At first glance, I thought it was the terrain provider.